src/Dialogs/BulkRenamer/Renamer.vala

```diff
@@ -59,7 +58,7 @@ public class Files.Renamer : Object {
             if (dir == directory) {
                 f.ensure_query_info ();
                 var row = new RenamerListRow (f);
-                // listbox.add (row);
+                listbox.insert (row, -1);
```

Comment:
seems like a more verbose way than just `append`?
```suggestion
                listbox.append (row);
```

Reply:
This way was originally used as a Gtk3 compatible way to be pushed to the `main` branch but yes, here we can go full Gtk4.